app/containers/ActionView/index.js

```diff
@@ -48,8 +49,15 @@ export class ActionView extends React.PureComponent { // eslint-disable-line rea
     }))
   )
 
+  renderRecommendations = (recommendations) => (
+    Object.values(recommendations).filter((recommendation) => recommendation.connected).map((recommendation) => ({
```

Comment:
then (see below) we also do not need to filter for connected recs as we are only getting connected recs to begin with